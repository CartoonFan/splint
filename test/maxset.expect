
Finished LCLint checking --- no code errors found

maxsetnoannotations.c: (in function noancopy)
maxsetnoannotations.c:4:1: Possible out-of-bounds store.  Unable to resolve
    constraint:
    requires: : maxSet ((a @ maxsetnoannotations.c:4:9 ) )  >=  maxRead ((b @
    maxsetnoannotations.c:4:11 ) )
     needed to satisfy precondition:
    requires: : maxSet ((a @ maxsetnoannotations.c:4:9 ) )  >=  maxRead ((b @
    maxsetnoannotations.c:4:11 ) )
     derived from strcpy precondition: requires: :
    maxSet ((<parameter 1> ) )  >=  maxRead ((<parameter 2> ) )

Finished LCLint checking --- 1 code error found
