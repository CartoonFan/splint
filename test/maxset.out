
Finished LCLint checking --- no code errors found
/a/apollo/af9/drl7x/re/LCLintDev/test/maxset

maxsetnoannotations.c: (in function noancopy)
maxsetnoannotations.c:4:1: Unresolved constraint:
    Lclint is unable to resolve Requires: : MAXSET ((a @ maxsetnoannotations.c:4
    :9 ) )  >=  MAXREAD ((b @ maxsetnoannotations.c:4:11 ) ) needed to satisfy
    Requires: : MAXSET ((a @ maxsetnoannotations.c:4:9 ) )  >=  MAXREAD ((b @
    maxsetnoannotations.c:4:11 ) )

Finished LCLint checking --- 1 code error found
